pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1521,6 +1521,11 @@ int tracepoint__sched__sched_process_exit(struct bpf_raw_tracepoint_args *ctx)
     if (!init_program_data(&p, ctx, SCHED_PROCESS_EXIT))
         return 0;
 
+    // the caller might not be an exit like syscall, but a signal handler
+    if ((p.event->context.syscall < 0) &&
```

Comment:
Gonna see tomorrow if checking against expected syscalls is a better solution, since I suspect that we can have syscall wrongly populated as positive as well.